README.md

```diff
@@ -76,16 +91,14 @@ The following tutorials will give you a sense of how to upgrade ASP.NET, Windows
 
 Learn more about upcoming support for Xamarin.Forms to .NET MAUI upgrades [here](docs/maui_support.md). 
 
-Download this free e-book on [Porting existing ASP.NET apps to .NET Core](https://aka.ms/aspnet-porting-ebook)
-
-[![Porting existing ASP.NET apps to .NET Core by Steve "ardalis" Smith](https://user-images.githubusercontent.com/782127/145730776-4f05fecf-01c3-4587-9e68-b45b42d83e92.png)
+Download this free e-book on [Porting existing ASP.NET apps to .NET Core](https://aka.ms/aspnet-porting-ebook).
 
 ## Installation
 
 ### Prerequisites
 
-1. This tool uses MSBuild to work with project files. Make sure that a recent version of MSBuild is installed. An easy way to do this is to [install Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
-2. This tool requires that your project builds. This may include installing [install Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) to ensure build SDKs (such as for web applications, etc) are available.
+1. This tool uses MSBuild to work with project files. Make sure that a recent version of MSBuild is installed. An easy way to do this is to [install Visual Studio 2022](https://visualstudio.microsoft.com/downloads/).
+2. This tool requires that your project builds. This may include installing [install Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) to ensure build SDKs (such as for web applications, etc) are available.
```

Comment:
Didn't realize the actual URLs wouldn't change. Might be worth just removing the year from the link text so we don't have to keep chasing this in the future :)
```suggestion
1. This tool uses MSBuild to work with project files. Make sure that a recent version of MSBuild is installed. An easy way to do this is to [install Visual Studio](https://visualstudio.microsoft.com/downloads/).
2. This tool requires that your project builds. This may include installing [install Visual Studio](https://visualstudio.microsoft.com/downloads/) to ensure build SDKs (such as for web applications, etc) are available.
```